charts/prometheus-node-exporter/templates/serviceaccount.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
```

Comment:
Please, enter the new field in values.yaml assigning it a default value.

Reply:
@zeritti its already exists in values.yaml file. So I am just using it in the serviceaccount.yaml
https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/values.yaml#L292

Reply:
Indeed, apologies, I missed it.